packages/@aws-cdk/aws-apigatewayv2-authorizers/README.md

```diff
@@ -28,6 +28,37 @@ classified into Lambda Authorizers, JWT authorizers and standard AWS IAM roles a
 available at [Controlling and managing access to an HTTP
 API](https://docs.aws.amazon.com/apigateway/latest/developerguide/http-api-access-control.html).
 
+Authorizers, and scopes can either be applied to the Gateway (applied to all routes) or specifically for each route.
+
+The example below applies the authorizer to all routes.
+
+```ts
+const authorizer = new HttpJwtAuthorizer({
```

Comment:
We should add the parts about `defaultAuthorizers` and `NoneAuthorizer` to the README in the `apigatewayv2` module, and move the part about `defaultAuthorizationScopes` in this file but under the Jwt Authorizers section.

Reply:
Seems pretty strange to do that. Shouldn't all the info about authorizers be in one place? Seems strange to have this info between two modules

Reply:
You are correct. Maybe we should leave it all in one place.

Then, let's just keep it all here but organize it a little. state a couple of statement about the NoneAuthorizer before providing the code snippet. And move the scope info into the jwt section. Does that make sense?

Reply:
Definitely could use a reorg, was thinking maybe:

- general info on authorizers (types etc)
- how to add an authorizer to an API (default stuff)
- how to add it to a route (and override)
  - mention none authorized in here
- Jwt authorizers
  - user pool authorizer